README.md

```diff
@@ -1,24 +1,26 @@
-# Frobnicator (this is a template)
+[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-22041afd0340ce965d47ae6ef1cefeee28c7c493a6346c4f15d667ab976d596c.svg)](https://classroom.github.com/a/M0kyOMLZ)
+# JuweAppka
 
-## Authors
-- Andrzej Głuszak (@agluszak on GitHub)
-- Linus Torvalds (@torvalds on GitHub)
+## Autorzy
+- Błażej Wilkoławski (@Buarzej on GitHub)
+- Paulina Kubera (@c00kiepreferences on GitHub)
 
-## Description
-Frobnicator is going to be a platformer game similar to Super Mario Bros made using Korge game engine.
+## Opis
+JuweAppka to aplikacja mobilna przeznaczona na systemy Android i iOS będąca aplikacją festiwalu muzycznego "Juwenalia UW".
 
-## Features
-- map generator
-- shooting
-- enemy AI
-- game state saving and loading
-- scores
+## Funkcje
+- harmonogram koncertów i innych wydarzeń towarzyszących w dniach festiwalu
+- lista aktualnie ogłoszonych artystów oraz możliwość odsłuchania ich najpopularniejszych utworów bezpośrednio w aplikacji
+- funkcja kupna biletów (w serwisie zewnętrznym) i możliwość wgrania ich do aplikacji w postaci pliku PDF
```

Comment:
> funkcja kupna biletów (w serwisie zewnętrznym)

czyli otwarcie przeglądarki?

> możliwość wgrania ich do aplikacji w postaci pliku PDF

co przez to rozumiecie?

Reply:
Kupno biletów będzie odbywało się rzeczywiście przez przeglądarkę (wbudowany widok strony internetowej w aplikacji, nie uruchomienie linku w zewnętrznej przeglądarce).

Użytkownik po zakupie biletu będzie miał możliwość wgrania go do aplikacji („prześlij plik…”) w formacie PDF, a aplikacja go zapamięta i umożliwi szybkie otwarcie kodu QR zawartego w PDF-ie (lub po prostu: otwarcie PDF-a), tak żeby bilet był pod ręką.